### ISSUE TITLE
Error: Couldn't connect to accessibility bus

### ISSUE THREAD

**Issue by kaesetoast** [theblacklion/gtk-theme-variant-switcher#6]
Hey folks, thanks for this great switch :-)

I just upgraded to the beta2 release of eOS. Now the switcher won't start propperly anymore. I always get this error message:

```
** (switcher.py:5336): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-GEIxoFyXkT: Connection refused
```

**Comment by theblacklion** [comment]
Hi @kaesetoast,

I had this warning message all over the place with various apps in beta1 and it didn't seem to break the apps. But I cannot reproduce that on my machine - not getting that error message anymore for some time now.

Does it really not start properly or is this message just annoying you?

**Comment by kaesetoast** [comment]
No, unfortunately it doesn't work propperly. The message doesn't really bother me. Maybe this is even not related at all. Somehow the switch seems to start, as it adds a dropshadow to the text in the titlebar. But the dark background and the border does not get rendered.

Here's a screenshot to show you what I'm talking about:

![bildschirmfoto vom 2015-02-10 09 41 31](https://cloud.githubusercontent.com/assets/1102078/6124015/d202db4a-b109-11e4-93d0-b892340f00b5.png)

**Comment by anatooly** [comment]
As for me too, like @kaesetoast on PhpStorm, after update system to Freya Beta 2.

And this add please to gscheme.xml 
+16         ("brackets","dark","Brackets")¬

**Comment by kaesetoast** [comment]
Seems like they are handling something differently with the GTK stuff in Beta2 then...

@anatooly you could just submit a PR for this change..

**Comment by theblacklion** [comment]
Yeah, I really hoped that I was the only one having this stupid problem. For me it looks like the eOS guys have broken the dark theme for GTK2 apps. I tried to switch to prefer the dark theme overall in the tweak tool and could see that the borders of Sublime, Chrome etc are broken. And my switcher doesn't do anything different - it just sits there and tries to set the preferred border theme variant.

I will try to post something in the communities and if that doesn't help, I'll post a bug report on launchpad. I'll post the links to them here. Would be nice if you could vote for them so that this is getting some proper attention.

For now I downgraded my theme package to:
elementary-theme_4.0-0~r410+pkg296~ubuntu0.3.1_all.deb
Everything after that (e.g. >r410) seems to break the dark theme variant for GTK2 apps.
I could install it using this:
sudo apt-get install elementary-theme='4.0-0~r410+pkg296~ubuntu0.3.1'
The place on my machine is:
/var/cache/apt/archives/elementary-theme_4.0-0~r410+pkg296~ubuntu0.3.1_all.deb

I will keep this issue open so that others can see the progress and that it's not realy the switchers fault but rather that the theme is currently broken. And we can expect to have it fixed soon - eOs is still in beta, so...

**Comment by theblacklion** [comment]
@kaesetoast @anatooly Does the workaround work for you?

**Comment by kaesetoast** [comment]
I didn't want to downgrade, so for now I'll live without dark borders. But thanks for the hint. I hope the eOS folks will fix this. For now that's what you get for running beta software in production :-)

**Comment by theblacklion** [comment]
Created a bug report. Hope they will address it.

https://bugs.launchpad.net/egtk/+bug/1421201

**Comment by theblacklion** [comment]
@kaesetoast @anatooly They fixed it already :). You can update the elementary-theme package to rev 414 and it should be fine again.
